package.json

```diff
@@ -10,7 +10,7 @@
   ],
   "description": "High bit depth, high resolution, many channels, low overhead",
   "scripts": {
-    "start": "vite",
+    "start": "vite --force",
```

Comment:
Unless you are consistently editing `node_modules`, I would remove this flag. Is there a reason you added it by default?